simply-rets-api-helper.php

```diff
@@ -1373,6 +1394,10 @@ public static function srResidentialResultsGenerator( $response, $settings ) {
                     $compliance_markup
                   </div>
                 </div>
+                <div class="sr-screen-reader" itemprop="geo" itemscope itemtype="http://schema.org/GeoCoordinates">
+                  <meta content="$lat" itemprop="latitude">
+                  <meta content="$lng" itemprop="longitude"
```

Comment:
The main reason is purely for better structured markup for search engines. I'm not aware of any other benefits. We just work to implement schemas on all of our content types within the sites that we build. 

As long as it is an HTML5 doctype then we don't need the closing meta tags.